ros/src/waypoint_updater/waypoint_updater.py

```diff
@@ -125,6 +125,8 @@ def distance_sqr(self, x0, y0, x1, y1):
         return (x0 - x1) ** 2.0 + (y0 - y1) ** 2.0
 
     def find_nearest_wp(self, x, y):
+        if getattr(self, 'waypoints') is None:
```

Comment:
This change is unnecessary – `waypoints` is initialized to an empty list in the class constructor, so the attribute will always exist, and never be set to `None`.

Reply:
Hi Helio, I would have to dive deep to the system to investigate root cause. This change was made after `None` exception was observed during onsite test. For simulator solution, this change is unnecessary.